### DISCUSSION TITLE
wyze pan cam v3 clon

### DISCUSSION THREAD

**Discussion by cristianmc** [gtxaspec/wz_mini_hacks#524] [Q&A]
hi ppl, i have a clon cam pan v3 provided by a telecom company, link for details :
https://tienda.personal.com.ar/accesorios/smarthome/smarthome-personalcam-blanco/d
physically and in specifications it is the same as the wyze pan cam v3, it comes with firmware 3.17.0.30 and it is adopted by an android app (another wyze app clone).
I would like to know if this project is compatible with the camera, I need to enable the rstp to adopt through zoneminder

**Comment by arguser** [comment]
@cristianmc Hola loco

As you might have already see, I was trying to troubleshoot this case on https://github.com/mnakada/atomcam_tools/issues/74 without success. I've provided @gtxaspec with the boot-loader so maybe there will be some progress for this model in the future?
Anyway, I just wanted to tell you not to upgrade firmware, hope you didn't do it already. Maybe there's a way to back it up and reverse updates in case they forbid mods to function.

**Reply by cristianmc** [reply]
Hello, I'm following all the repos haha.
Unfortunately, I did update the firmware of one camera, but I have another one that should arrive these days, I have friends who also updated but some of their cameras already came with updated firmware

**Reply by cristianmc** [reply]
News, I have a camera with firmware 3.17.0.47 (seems to be the latest for the personal model) and another with firmware 3.17.0.30

**Reply by arguser** [reply]
Have you tried `atomcam_tools`on both?

**Reply by cristianmc** [reply]
not yet, the one with the old firmware has just arrived, when I finish my work I'll try it

**Reply by amateuramo** [reply]
hola cristian, la app de personal funciona bastante mal (de hecho es una beta), quiero adaptarla para usarla via rstp y monitorearla directo. estoy en la misma. cualquier avance/novedad, será agradecida. por lo que leí tocar el firmware puede ser una lotería y hasta ahora no quiero hacerlo. vi que con el firmware de la wyze se podria acceder via wyze y usar un repositorio que se agrega al home assistant. pero hay que flashearla y no se si es posible. no he averiguado aun si se puede hacer un backup del firmware original para poder restaurarla en caso de brickeo. cualquier novedad te pido me avises, gracias!

**Reply by gtxaspec** [reply]
do you have a link to the firmware for your model camera? or can someone post iCamera binary for analysis.

**Reply by arguser** [reply]
Hola @amateuramo hasta ahora creo que soy el que más experimento con esto, mucho de lo que fui probando podes verlo en este issue https://github.com/mnakada/atomcam_tools/issues/74 también tengo [un hilo en Twitter](https://twitter.com/arguser/status/1658214242149867537) donde fui registrando mi experiencia desde que tengo las cámaras.

Hey @gtxaspec I would gladly provide a firmware but I'm not quite sure how to fetch it. I've uploaded the binary here https://github.com/escapingnetwork/campanario/tree/TELEAR_CamPan/iCamera

@gtxaspec to extend info, I've been trying `atomcam_tools` the `iCamera_app` crashed if provided a `callback` lib. I tried with both libs you posted on this repo too.

**Reply by arguser** [reply]
@gtxaspec I've received a Firmware which I uploaded here https://github.com/escapingnetwork/campanario/tree/TELEAR_CamPan/Firmwares 

Hope you can take a look into it and please hmu if you need anything.

**Comment by gtxaspec** [comment]
i'll take a look soon for you

**Reply by arguser** [reply]
@gtxaspec there has been progress https://github.com/mnakada/atomcam_tools/issues/74#issuecomment-1574971242

**Reply by matlos74** [reply]
Pudieron porner a la camara de personal el firmware original de Wyze?

**Reply by arguser** [reply]
Hola, en https://github.com/escapingnetwork/campanario encontrás la version que anda. No es el firmware de wyze es un mod de atomcam.

**Comment by gjmv** [comment]
Hola, pudieron volver al firmware de wyze? 
O habilitar los puertos?
No puedo acceder ni por http, solo con la app de personal, que es muy limitada.
Saludos

Edit: tenia problemas con el formato de la memoria, ahora logré acceder a la web.
Igual me gustaría volver al firmware stock, pero no se si es seguro.

**Reply by arguser** [reply]
Hola, en https://github.com/escapingnetwork/campanario encontrás la version que anda. No es el firmware de wyze es un mod de atomcam. Esto no modifica el firmware de la cámara si no que lo extiende, si querés volver al stock solo tenes que quitar el contenido de la tarjeta.

**Comment by locomax16** [comment]
Buenas, hay algún update del mod o firmware liberado?

**Reply by endertable** [reply]
Can this even be purchased from USA?

**Reply by locomax16** [reply]
I understand that this particular video surveillance product is only sold in Argentina since the company that markets it is local. The camera model used apparently would be:
**"Wyze Cam Pan V3" or its Chinese clone equivalent "Atom Cam Swing"**

**Comment by gtxaspec** [comment]
hola amigos,
ya supportamos personalcam en thingino!

https://github.com/themactep/thingino-firmware
https://thingino.com
check it out!

**Reply by choncba** [reply]
Looks Awesome! 
I have a Wyze Pan V3 clone camera, not clear how to get this firmware running on this, looks like there is several methods to flash, I tried sd card, but with no luck. There is some guide for this model coming from diferent firmware? 
Regards.-

**Reply by gtxaspec** [reply]
join the thingino telegram or discord, we can help!
